keyboards/churrosoft/deck8/rgb/info.json

```
@@ -0,0 +1,27 @@
+{
+  "keyboard_name": "Deck-8 RGB",
+  "usb": {
+    "pid": "0xC101"
+  },
+  "ws2812": {
+    "pin": "GP17",
+    "driver": "vendor"
+  },
+  "features": {
+    "rgb_matrix": true
+  },
+  "rgb_matrix": {
+    "driver": "WS2812",
+    "max_brightness": 200,
+    "layout": [
+      {"flags": 4, "matrix": [0, 0], "x": 0, "y": 0},
+      {"flags": 4, "matrix": [0, 1], "x": 1, "y": 0},
+      {"flags": 4, "matrix": [0, 2], "x": 2, "y": 0},
+      {"flags": 4, "matrix": [0, 3], "x": 3, "y": 0},
+      {"flags": 4, "matrix": [1, 0], "x": 0, "y": 1},
+      {"flags": 4, "matrix": [1, 1], "x": 1, "y": 1},
+      {"flags": 4, "matrix": [1, 2], "x": 2, "y": 1},
+      {"flags": 4, "matrix": [1, 3], "x": 3, "y": 1}
```

```suggestion
      {"flags": 4, "matrix": [0, 0], "x":  45, "y": 21},
      {"flags": 4, "matrix": [0, 1], "x":  90, "y": 21},
      {"flags": 4, "matrix": [0, 2], "x": 135, "y": 21},
      {"flags": 4, "matrix": [0, 3], "x": 180, "y": 21},
      {"flags": 4, "matrix": [1, 0], "x":  45, "y": 42},
      {"flags": 4, "matrix": [1, 1], "x":  90, "y": 42},
      {"flags": 4, "matrix": [1, 2], "x": 135, "y": 42},
      {"flags": 4, "matrix": [1, 3], "x": 180, "y": 42}
```

The physical x and y positions of LEDs are limited to a range of [224 by 64](https://docs.qmk.fm/#/feature_rgb_matrix?id=common-configuration). You can consider scaling the LEDs in this manner to provide sufficient breadth for RGB animation.

---

Sweet. Just tested these changes with the spash animation and it's way better